keyboards/phrygian/ph100/boards/GENERIC_STM32_F042X6/board.c

```diff
@@ -0,0 +1,268 @@
+/*
```

Comment:
All of these chibiOS files may not be needed, since they also reside in `/platform/chibios/GENERIC_STM32_F04X6`